### PR TITLE
Dev deps bump PR - first one in 2020 edition

### PR DESCRIPTION
**Pull request by etpinard**
This PR bumps two "dependencies":
- `d3-interpolate` from v1 to to v1.4.0,
- `d3-hierarchy` from v1.1.8 to v1.1.9,

In addition some dev-dependencies are updated in this PR. 

@archmoj we now have

image

So, our outdated deps are:

- `d3` can't bump until a plotly.js v2
- `d3-force` changes some sankey baselines, needs investigation
- `es6-promise` can't bump as this breaks plotly.js in IE11 (if I recall correctly)
- `fs-extra`, could bump but requires no-trivial changes in `tasks/`
- `image-size`, can't bump as newer versions drop support for the Node version used in our image-test docker container
- `jsdom`, could bump but requires no-trivial changes in `tasks/`
- `mapbox-gl`, could bump, but I like to bump `mapbox-gl` in mapbox-related PRs
- `regl`, @archmoj what's your take on that one?
- `topojson-client`, the newer version appear to not perform as well when I tested it in some past geo work, we could try again.

### CODE REVIEW

**Comment by etpinard** [comment]
@archmoj as I'm about to go offline for two weeks, please go ahead and merge this PR if you approve it.

File: package.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
     "mouse-change": "^1.4.0",
     "mouse-event-offset": "^3.0.2",
     "mouse-wheel": "^1.2.0",
-    "ndarray": "^1.0.18",
+    "ndarray": "^1.0.19",
```
Comment (archmoj) [inline package.json]: This is support for `BigInt64Array` types and 5 days old.
I prefer opening pull requests for the modules listed below:
```
ndarray-pack
ndarray-scratch
simplicial-complex-contour
gl-contour2d
gl-cone3d
gl-line3d
gl-mesh3d
gl-scatter3d
gl-plot2d
gl-plot3d
gl-select-static
gl-buffer
gl-texture2d
vectorize-text
```

Comment (archmoj) [reply package.json]: Reverted the `ndarray` version in https://github.com/plotly/plotly.js/pull/4475/commits/d182131d7bb513a61e3b259434bcaaf8e369f3b6 for now.
We could bump that one when the following PRs are merged; and then we could patch gl-vis modules as well.
https://github.com/mikolalysenko/vectorize-text/pull/16
https://github.com/scijs/ndarray-scratch/pull/7
https://github.com/scijs/ndarray-pack/pull/6
https://github.com/mikolalysenko/simplicial-complex-contour/pull/1

Comment (etpinard) [reply package.json]: Good call. Thanks @archmoj !

**Pull request merged**